packages/webawesome/docs/docs/resources/changelog.md

```diff
@@ -1,10 +1,11 @@
 ---
 title: Changelog
+date: git Last Modified
 description: Changes to each version of the project are documented here.
 layout: page-outline
 ---
 
-<p class="wa-caption-s">Last updated: <wa-format-date month="long" day="numeric" year="numeric" date="{{ lastUpdatedISO }}"></wa-format-date></p>
+<p class="wa-caption-s">Last updated: <wa-format-date month="long" day="numeric" year="numeric" date="{{ page.date }}"></wa-format-date></p>
```

Comment:
I _believe_ you’ll want `{{ page.date.toISOString() }}` per the docs for `wa-format-date`: https://webawesome.com/docs/components/format-date/#attributes-and-properties